### PR TITLE
Introduce the virtualpdu entry point

### PR DESCRIPTION
**Pull request by idjaw**

### CODE REVIEW

File: virtualpdu/main.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from __future__ import print_function
```
Comment (stephanerobert) [inline virtualpdu/main.py]: This is not needed. Print as a function is supported since py26 and I don't think we will be going that far back.

Comment (idjaw) [reply virtualpdu/main.py]: Agreed. Removing.

File: virtualpdu/tests/integration/test_entrypoint.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# limitations under the License.
+import subprocess
+
+import sys
+
+import os
+from virtualpdu.tests import base
+
+
+def get_entrypoint_path(entrypoint):
```
Comment (fbouliane) [inline virtualpdu/tests/integration/test_entrypoint.py]: nit; you can ignore. is it the good scope for this ?
I would add it as a private method of TestEntrypointIntegration

File: virtualpdu/tests/integration/test_entrypoint.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    return os.path.join(os.path.dirname(sys.executable), entrypoint)
+
+
+class TestEntrypointIntegration(base.TestCase):
+    def test_entry_point_returns_hello_world(self):
```
Comment (fbouliane) [inline virtualpdu/tests/integration/test_entrypoint.py]: it does not return hello world. it prints VirtualPDU Entry Point ;-)

Comment (idjaw) [reply virtualpdu/tests/integration/test_entrypoint.py]: Good catch. Forgot to rename that

File: virtualpdu/tests/integration/test_entrypoint.py (hunk 2 above)
Comment (idjaw) [reply virtualpdu/tests/integration/test_entrypoint.py]: Yeah. Let's do that.

**Comment by fbouliane** [comment]
:+1:

**Comment by stephanerobert** [comment]
👍

**Pull request merged**